doc/apple2.sgml

```diff
@@ -92,7 +92,28 @@ several useful settings:
 
 </descrip><p>
 
+<descrip>
+
+  <tag/Installing your code in the Language Card/
+  The code in the LC segment will automatically be copied from the end of the
+  binary into the Language Card at startup, even if the segment is empty (a
+  0-byte copy). This allows for tighter code. This code will be copied using the
+  BLTU2 AppleSoft function that is available starting with the Apple II plus,
+  but not in the original 48kB Apple II with an Integer BASIC ROM. This shows as
```

Comment:
I don't think that `48kB` is relevant here. The Apple ][+ is totally the same regarding RAM amount considerations as the Apple ][.